lib/data/store.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:convert';
+
+import 'package:shared_preferences/shared_preferences.dart';
+
+class Store {
+
+  static Future<bool> saveString(String key, String value) async {
+    final prefs = await SharedPreferences.getInstance();
+    return prefs.setString(key, value);
+  }
+
+  static Future<bool> saveMap(String key, Map<String, dynamic> value) async {
+    return saveString(key, jsonEncode(value));
+  }
+
+  static Future<String> getString(String key, [String defaultValue = '']) async {
+    final prefs = await SharedPreferences.getInstance();
+    return prefs.getString(key) ?? defaultValue;
+  }
+  static Future<Map<String, dynamic>> getMap(String key) async {
```

Comment:
Format the document and don't forget the documentation

Reply:
added requested documentation